src/gf2n.h

```diff
@@ -3,15 +3,39 @@
 
 #include "gf.h"
 
+enum MulType { mul_log_tab, split_8_8 };
+enum DivType { div_log_tab, div_by_inv };
+enum InvType { inv_by_div, inv_ext_gcd };
+
```

Comment:
Can you put enum values in upper case, e.g. MUL_LOG_TAB and inside the class definition ?